tools/st2_deploy.sh

```diff
@@ -465,6 +465,8 @@ install_apt() {
   # Install packages
   echo "Installing ${APT_PACKAGE_LIST}"
   apt-get install -y ${APT_PACKAGE_LIST}
+  # Now that pip is installed set PIP=`which pip` again.
+  PIP=`which pip`
```

Comment:
Also, ideally you would put it immediately after the line where we upgrade pip otherwise it's not really clear why it's here / needed.